estela-api/api/views/stats.py

```diff
@@ -287,19 +315,18 @@ def list(self, request, *args, **kwargs):
         spider = Spider.objects.get(sid=kwargs["sid"])
         jobs_set = spider.jobs.filter(created__range=[start_date, end_date])
 
-        job_collections_names: List[str] = [
+        job_stats_ids: List[str] = [
```

Comment:
Same here, we can create a ticket to use this notation in the entire repo.

Reply:
Ok I think is a good idea, also to start using best practices, I'll create a ticket.

Reply:
Awesome! Thank you @webtaken